src/matlab2tikz.m

```diff
@@ -2979,11 +2964,10 @@ case size(cData,2) % specific y-data is given
             % Make sure this happens exactly *once*.
 
             if ~m2t.barAddedAxisOption
-                bWFactor = get(h, 'BarWidth');
+                BarWidth = get(h, 'BarWidth');
```

Comment:
The switch case introduces a separation of the workflow but the code retains the same scope, i.e. basically deals with bar width and shift. So, I would not change naming except for the lower case.